pkg/provider/cloud/openstack/helper.go

```diff
@@ -135,7 +135,7 @@ func validateSecurityGroupsExist(netClient *gophercloud.ServiceClient, securityG
 func deleteSecurityGroup(netClient *gophercloud.ServiceClient, sgName string) error {
 	results, err := getSecurityGroups(netClient, ossecuritygroups.ListOpts{Name: sgName})
 	if err != nil {
-		return fmt.Errorf("failed to get security group: %v", err)
+		return fmt.Errorf("failed to get security group: %w", err)
```

Comment:
You should alsouse `%v` in `getSecurityGroups`

Reply:
done